openedx/core/djangoapps/api_admin/utils.py

```diff
@@ -11,5 +13,5 @@ def course_discovery_api_client(user):
     course_discovery_client = Client.objects.get(name=CLIENT_NAME)
     return EdxRestApiClient(
         course_discovery_client.url,
-        jwt=get_id_token(user, CLIENT_NAME)
+        jwt=get_id_token(user, CLIENT_NAME, secret_key=settings.JWT_AUTH['SECRET_KEY'])
```

Comment:
Huh, I didn't even realize that was a thing. Should we be using `get_value` everywhere instead of accessing `settings` directly?